crates/atuin-common/i18n.toml

```diff
@@ -0,0 +1,13 @@
+# (Required) The language identifier of the language used in the
+# source code for gettext system, and the primary fallback language
+# (for which all strings must be present) when using the fluent
+# system.
+fallback_language = "en-US"
```

Comment:
part of me would love to force en-GB 😏

Reply:
I'll be honest, it was en-GB until a commit or two ago, so ... :D 

That said, it may be an gentle way to encourage a wide sample, as more people will spot that is falling back to -ise spellings instead of correctly picking up US locales, than the other way around 🤔